### PR TITLE
Combobox: add tests for backspace

### PR DESCRIPTION
**Pull request by spectranaut**
This PR is https://github.com/w3c/aria-practices/pull/1363 with some code review comments implemented and master merged into the branch and conflicts resolved!

### CODE REVIEW

**Review by jugglinmike — CHANGES_REQUESTED** [review]
`npm run regression-report` is failing on the `master` branch. This makes it harder to understand if the pull request is regressing. It turns out that this one is not, so the problem is out of scope for this patch. I'm mentioning it anyway in hopes that it gets on someone's radar.

I've verified that the tests pass. I could probably give more input, but I'm out of time for the day!

File: test/tests/combobox_autocomplete-both.js [hunk 1]
```diff
@@ -541,6 +540,198 @@ ariaTest(
...
+    );
+
+    // Send key ARROW_DOWN
+    await t.context.session
+      .findElement(By.css(ex.textboxSelector))
+      .sendKeys(Key.ARROW_DOWN);
+
+    // Confirm that the listbox is still open
+
+    await assertAttributeValues(t, ex.textboxSelector, 'aria-expanded', 'true');
```
[the remark this answers is not in the document]
Comment (spectranaut) [reply test/tests/combobox_autocomplete-both.js:619]: Thanks for that catch, @jugglinmike! fixed :)

**Comment by spectranaut** [comment]
hi @mcking65 -- This PR was originally @zcorpan work so I reviewed it when I adopted it, and Mike took a look (though he only got part way through).  I think it's good to go, in that it has been looked at by me, @zcorpan and @jugglinmike

**Comment by mcking65** [comment]
Hurray, someone fixed the W3C payment plan issue with Travis and our Travis no-op job is passing again!

**Pull request merged**